### ISSUE TITLE
Can not disable Global Media Control any longer

### ISSUE THREAD

**Issue by Banderi** [Alex313031/thorium#657]
**System Details**
 - OS: Windows 10
 - Thorium Version 123.0.6312.133 64-bit

**Problem**
I used to use the `--disable-features=GlobalMediaControls` flag on the startup shortcut to disable the Global Media Controls panel. Unfortunately, this no longer seems to work with the recent update. The `#global-media-controls-updated-ui` flag in the settings (and any other) also does not remove the button.

**Screenshots**
![image](https://github.com/Alex313031/thorium/assets/16541079/5392078e-5371-4aef-ba22-8934c449eb13)

**Additional Notes**
I've tried to browse through the source for more insight on how the `--disable-features=` flag is parsed, and understand why it no longer works, however I can't seem to find anything regarding this in the source. I see some sporadic references to `kEnableFeatures` but that also doesn't seem actually used anywhere. The only file where that is used is a browser test .cc that makes references to various missing files such as `content/shell/common/shell_switches.h`. What is going on here?

**Comment by gz83** [comment]
Does the 123 version of Chrome or Chromium have the same problem?

**Comment by Alex313031** [comment]
@Banderi Try also disabling GlobalMediaControlsUpdatedUI

**Comment by Banderi** [comment]
> @Banderi Try also disabling GlobalMediaControlsUpdatedUI

I tried disabling that flag as well as `hardware-media-key-handling` and `enable-web-app-system-media-controls-win` and other combinations from comments I found on the net but I couldn't get it to work sadly.

Do you know perhaps where in the code the `--disable-features=` flags are parsed? I can't find it anywhere and the Chromium src search on github is bugged (stuck on indexing)

**Comment by Alex313031** [comment]
@Banderi Use the source code search for Chromium here > https://source.chromium.org/chromium

Most of these features are controlled by this file > https://source.chromium.org/chromium/chromium/src/+/main:media/base/media_switches.cc

**Comment by Banderi** [comment]
> @Banderi Use the source code search for Chromium here > https://source.chromium.org/chromium
> 
> Most of these features are controlled by this file > https://source.chromium.org/chromium/chromium/src/+/main:media/base/media_switches.cc

Ok, I think I understand my confusion now. I couldn't figure out why the searches weren't returning anything, and after looking through the source I couldn't figure out why there was whole sections missing from the folders. Now I realize that this fork doesn't include the Chromium source in its entirety, and requires third party tools to fetch the rest (depot_tools).
Also, there isn't a single file that tailors and lists all the feature flags recognized by Chromium as I initially assumed, instead the `BASE_FEATURE(...)` command is invoked in the files that pertain to those features (e.g. `kGlobalMediaControls` inside `media_switches.cc`) and there it is initialized and recorded to the engine. (This command and its internals are defined in the Chromium source but not in the fork since they haven't been touched.)

I'm gonna try to look now at the source directly now and see if anything changed with the usage of the `kGlobalMediaControls` flag and why it's not working on my end...

**Comment by Nicols0Mart** [comment]
I'm using latest build from today and this is still broken on windows release. Disabling the flag #global-media-controls-updated-ui does not remove the icon when playing a video in YouTube for example

**Comment by Alex313031** [comment]
@Nicols0Mart @Banderi Yeah, it still doesn't seem to be able to be disabled, even with `--disable-features=GlobalMediaControls`

**Comment by jordanlopez8098** [comment]
jjj

**Comment by gz83** [comment]
Close stale and inactive issues.
